ui/scripts/docs.js

```diff
@@ -1008,7 +1008,7 @@ cloudStack.docs = {
     },
     // Add volume
     helpVolumeName: {
-        desc: 'Give the volume a unique name so you can find it later.',
+        desc: 'Give a unique volume name. If it is not provided, a name will be generated randomly.',
```

Comment:
did notice this until I started working on a similar issue for createNetwork. Shouldn't this be in a dictionary somewhere? @milamberspace You are the expert. Can you give an opinion?